create_cards/views.py

```diff
@@ -234,7 +234,7 @@ def create_card_save(request, required_key, event_name):
                 card_info["cost"] = int(card_info["cost"])    
             if "effects" in card_info:
                 for e in card_info["effects"]:
-                    if "amount" in e:
+                    if "amount" in e and e["amount"] != None:
```

Comment:
This line change was unrelated but fixed a bug in the card builder.

Reply:
why not e.get("amount") is not None ?